frontend/src/v5/ui/routes/dashboard/projects/drawings/drawingsList/drawingsListItem/drawingsListItem.component.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
```

Comment:
When "last updated" is used for sorting, items with not last update are always first. That behaviour is different for containers/feds 

https://github.com/3drepo/3drepo.io/assets/43547703/3da30128-22c7-4db3-9275-8289189d298b